client/galaxy/style/scss/theme/blue.scss

```diff
@@ -71,15 +71,15 @@ $font-family-sans-serif: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
     "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
 $font-family-monospace: Monaco, Menlo, Consolas, "Courier New", monospace;
 $font-family-base: $font-family-sans-serif;
-$font-size-base: 0.8rem; // default 1rem
```

Comment:
@guerler Can we leave these comments in (and the ones below), for future documentation?  It helps (at least me) remember we're downsizing from the bootstrap defaults, and by how much.

Reply:
Sure.